arkane/ess/gaussian.py

```diff
@@ -398,9 +398,9 @@ def load_scan_energies(self):
             vlist = vlist[:-1]
 
         # Determine the set of dihedral angles corresponding to the loaded energies
```

Comment:
Do any of these require similar modifications?
https://github.com/ReactionMechanismGenerator/RMG-Py/blob/627777034e82462f0570b93b458d65c547adc6e9/arkane/ess/qchem.py#L307
https://github.com/ReactionMechanismGenerator/RMG-Py/blob/fe6e69a94afcf40220e4980b0b772029f66fb571/arkane/ess/terachem.py#L275

Reply:
I think there is no `_load_scan_angle` method for qchem and terachem logs. I have no experience on these two packages and prefer not to change these methods in the current PR.